python/Dockerfile

```diff
@@ -1,8 +1,23 @@
-FROM alpine
+# docker image build -t build2023/app:latest -f python/Dockerfile .
+# docker run -d -p 9000:9000 build2023/app:latest
+
+FROM python:3.10.6-alpine
 
 RUN addgroup -S mercari && adduser -S trainee -G mercari
-# RUN chown -R trainee:mercari /path/to/db
+
+WORKDIR /app
+
+COPY python python 
+COPY db db
+
+COPY /python/requirements.txt .
+RUN pip install --no-cache-dir --upgrade -r requirements.txt
```

Comment:
nit: it looks like the file is copied to the container as part of line 10. So it looks like you can try specifying this path directly to `pip install ... -r <path/to/requirements.txt>`